Add Zimi Cloud Controller Documentation

mhannon11 opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->

Add documentation for new integration for Zimi Cloud Connect support.


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/129876
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [ X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Introduced comprehensive documentation for the Zimi Cloud Controller integration with Home Assistant, detailing functionality and configuration.

- **Documentation**
	- Added troubleshooting guidance for common issues, including device discovery and authorization errors.
	- Included instructions for configuring the integration and managing Zimi-based devices.
	- Specified supported and unsupported devices for clarity.
	- Explained the data update mechanism and noted known limitations.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

klaasnicolaas committed:
Rephrase removal section for multiple integrations (#36093)

klaasnicolaas committed:
Fix indentation in configuration across multiple integrations (#36090)

* Fix indentation in configuration across multiple integrations

* Forgot to commit this one

klaasnicolaas committed:
Fix url to voice troubleshooting (#36088)

klaasnicolaas committed:
Remove configuration keys from docs (#36089)

klaasnicolaas committed:
Convert to details block for troubleshootings (#36091)

klaasnicolaas committed:
Add known limitation section to Autarco integration (#36092)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump webrick from 1.9.0 to 1.9.1 (#36094)

dependabot[bot] committed:
build(deps-dev): bump textlint from 14.3.0 to 14.4.0 (#36096)

dependabot[bot] committed:
build(deps-dev): bump regexp_parser from 2.9.2 to 2.9.3 (#36095)

zweckj committed:
Update lamarzocco.markdown (#36098)

joostlek committed:
Remove Spotify sensors (#36099)

CoMPaTech committed:
Plugwise Add update and removal information (#36055)

Commit:
Update cisco_webex_teams.markdown (#36100)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11670 to 0.5.11672 (#36114)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump nokogiri from 1.16.7 to 1.16.8 (#36115)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump logger from 1.6.1 to 1.6.2 (#36116)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sass-embedded from 1.81.0 to 1.81.1 (#36117)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

robji committed:
Update samsungtv.markdown (#36112)

frenck committed:
Merge branch 'current' into rc

frenck committed:
2024.12: Fix nasweb version

frenck committed:
2024.12: Update dev blogs

frenck committed:
2024.12: Update changelog

Madelena committed:
Fancy homepage (#35945)

frenck committed:
Merge branch 'current' into rc

jbouwh committed:
Fix typo in mqtt docs (#36129)

frenck committed:
Merge branch 'current' into rc

jlpouffier committed:
Update Assist section for 2024.12 release (#36045)

Co-authored-by: Michael Hansen <hansen.mike@gmail.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

frenck committed:
2024.12: Adjust date, title & description

frenck committed:
2024.12: Finalize backward incompatible changes

frenck committed:
2024.12: Python 3.13

frenck committed:
2024.12: Finish Python 3.13 section

frenck committed:
2024.12: Update voice section

frenck committed:
2024.12: Finish UoM translations

frenck committed:
2024.12: Finish IQS

frenck committed:
2024.12: IQS images

frenck committed:
2024.12: Finish other noteworthy changes

frenck committed:
2024.12: Tweaks

frenck committed:
2024.12: Other noteworthy integration improvements

frenck committed:
2024.12: Iterate on the voice section

frenck committed:
2024.12: Adjust image for inclusion in voice section

frenck committed:
2024.12: Finish scene section

frenck committed:
2024.12: Update social graph

frenck committed:
2024.12: Update faster voice YT vid

frenck committed:
2024.12: Add intro

frenck committed:
2024.12: Process review comments

frenck committed:
2024.12: Process review comments

frenck committed:
2024.12: Process review comments

frenck committed:
2024.12: Fix open graph

frenck committed:
2024.12: Small fixes

frenck committed:
2024.12: Small fixes

frenck committed:
Apply suggestions from code review

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

frenck committed:
2024.12: Small fixes

frenck committed:
Apply suggestions from code review

Co-authored-by: Martin Hjelmare <marhje52@gmail.com>

joostlek committed:
Fix blogpost (#36162)

arturpragacz committed:
Organize Onkyo sections (#36164)

rikroe committed:
Rework BMW docs for new quality scale (#36083)

* Rework BMW docs for new quality scale

* Fix typos

* Apply suggestions from code review

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Apply suggestions from code review

* Tiny tweaks

* markdown tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

RaHehl committed:
Fix UnifiProtect Integration Examples to Prevent Triggering on Doorbell Reboot (#36183)

mbbush committed:
Fix broken link (#36186)

laupalombi committed:
Voice docs quick fixes (#36173)

* fix link for voice in sitemap

* removed mention to table in voice_remote_local_assistant

* Custom sentences - moved changing config file via yaml to its own doc

mrunde committed:
Fix typo in Plant Monitor documentation (#36170)

Replace `date` with `data` in the "Data source" example.

agners committed:
Document Apple Thread border router update limitations (#36192)

* Document Apple Thread border router update limitations

Document that devices behind Apple Thread border routers cannot be
updated through Home Assistant.

* Update source/_integrations/matter.markdown

Co-authored-by: Marcel van der Veldt <m.vanderveldt@outlook.com>

* Apply suggestions from code review

---------

Co-authored-by: Marcel van der Veldt <m.vanderveldt@outlook.com>

drewsteinacher committed:
Remove extraneous backtick (#36196)

frenck committed:
2024.12.1 (#36197)

* 2024.12.1

* Fix date/version

Co-authored-by: Simon Lamon <32477463+silamon@users.noreply.github.com>

---------

Co-authored-by: Simon Lamon <32477463+silamon@users.noreply.github.com>

gjohansson-ST committed:
Add troubleshooting section to command_line (#36190)

ndrwrbgs committed:
Tiny update to switch.template.markdown (#36189)

dependabot[bot] committed:
build(deps-dev): bump google-protobuf from 4.29.0 to 4.29.1 (#36168)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rbs from 3.6.1 to 3.7.0 (#36169)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11672 to 0.5.11681 (#36167)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#36160)

Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

joostlek committed:
Document config flow setup in geniushub (#36048)

dependabot[bot] committed:
build(deps-dev): bump sass-embedded from 1.81.1 to 1.82.0 (#36143)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump json from 2.8.2 to 2.9.0 (#36142)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop from 1.69.0 to 1.69.1 (#36140)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

kristian committed:
Update vicare.markdown adding an API limitation note (#36176)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

dontinelli committed:
Update documentation of status values (#36105)

watercrossing committed:
Removed left over reference to relative_time() (#36150)

relative_time() was deprecated in favor of time_since() and time_until() in #111177

felipecrs committed:
Add unsupported bluetooth adapter Ugreen CM749 (#36151)

Passific committed:
Add missing discovery prefix on device tracker's MQTT topics (#36119)

Shutgun committed:
Add example automations to devolo Home Network (#36101)

mj23000 committed:
Add supported features section to Bang & Olufsen (#36171)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

thheller committed:
Update edl21.markdown (#36148)

frenck committed:
Hotfix gemlock

Commit:
Update cisco_webex_teams.markdown (#36203)

Commit:
Fix typo in kef integration docs (#36204)

CrazyVito11 committed:
Fixed typo on the HA Cloud voice page (#36210)

drewsteinacher committed:
Link to min_version docs where it is referenced (#36233)

SvenPittelkow committed:
Update voice_remote_local_assistant.markdown (#36232)

dependabot[bot] committed:
build(deps): bump nokogiri from 1.16.8 to 1.17.0 (#36245)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11681 to 0.5.11690 (#36246)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

thromer committed:
Update link to "Using blueprints" section of template documentation. (#36243)

ildar170975 committed:
Update template.markdown: make "configuration" a separate chapter (#36209)

Shutgun committed:
Update high-level description of devolo Home Control (#36248)

starkillerOG committed:
Add Reolink NVS8 and NVS16 as supported (#36252)

zweckj committed:
Update acaia.markdown (#36253)

AJediIAm committed:
Nordpool doc update (#36228)

* Update nordpool.markdown

- Use commonly used table format to list the sensors. 
- Elaborate on peak and off-peak timeframe. 
- Add an example of a UI template. 
- Update YAML template to match the UI screenshots (for consistency)
- Add an example of how to use the template sensor in the energy dashboard.

* Apply suggestions from code review

Remove whitespace

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>

* Create test.txt

* Add files via upload

* Add files via upload

* Update nordpool.markdown

do images properly

* Delete source/images/integrations/nordpool/test.txt

file not needed

* Tiny tweaks

---------

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>
Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

starkillerOG committed:
Add 3 day blinds app instructions (#36231)

* Add 3 day blinds app instructions

* Styling

* Update source/_integrations/motion_blinds.markdown

Co-authored-by: Simon Lamon <32477463+silamon@users.noreply.github.com>

---------

Co-authored-by: Simon Lamon <32477463+silamon@users.noreply.github.com>

kaffolder7 committed:
Update anthropic.markdown (#36255)

Terms for Anthropic API usage changed this week to allow individuals & hobbyists to utilize their API. Changed the legal note in the documentation to reflect this change.

stevengoossensB committed:
Update opentherm_gw.markdown - Change default port (#36249)

starkillerOG committed:
Include Reolink kit cameras as only compatible through the NVR (#36263)

bouwew committed:
Add Plugwise breaking-change info to 2024.12.0 release note (#36250)

mj23000 committed:
Add limitations section to Bang & Olufsen (#36172)

tr4nt0r committed:
Improve IronOS documentation (#36076)

* Improve IronOS documentation

* Remove full URL

* fix yaml

* remove

---------

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

Gitsarry committed:
Improve threshold integration documentation example comment (#36147)

mxr committed:
Clarify reference to icloud storage folder location (#36113)

Bre77 committed:
Update splunk.markdown (#36265)

ctalkington committed:
Improve Roku documentation (#36247)

* improve roku documentation

* Update roku.markdown

* Update roku.markdown

* Update roku.markdown

* Update roku.markdown

* Update roku.markdown

* Tiny tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

catsmanac committed:
enphase envoy docs quality improvement (#36128)

* enphase envoy docs quality improvement

* apply coderabbitai comments

* tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

farmio committed:
Update Fronius documentation for new quality scale rules (#36033)

* Update Fronius documentation

* add ecample

* simplify example

* remove copy&past relict

* further improvements

* fix link

* Tiny tweaks

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

cnotin committed:
Describe that conditions of an 'if' are 'and' (#36270)

mrdarrengriffin committed:
Add voice livestream card (#36278)

* Add voice livestream thumbnail

* Undo replaced image

* Add livestream thumbnail

* Add voice chapter 8 livestream card

* Improved target="_blank" issue

* Added aria-label

frenck committed:
2024.12.2 (#36283)

dependabot[bot] committed:
build(deps): bump nokogiri from 1.17.0 to 1.17.1 (#36287)

ccutrer committed:
Link to other home automation hubs with some support for MQTT discovery (#36286)

Shutgun committed:
Add removal instructions to devolo Home Control (#36288)

andrewsayre committed:
Clean up HEOS docs (#36281)

gjohansson-ST committed:
Updates to Nord Pool integration (#36261)

gjohansson-ST committed:
Add note on use of json_attribute in command_line (#36282)

laupalombi committed:
OpenAI tutorial with Assist edited to cover globally all LLM-based agents (#36251)

* AI options with Assist reworked

* Update assist_create_open_ai_personality.markdown

* Update assist_create_open_ai_personality.markdown

list fixes

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update assist_create_open_ai_personality.markdown

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* tiny  tweak

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: JLo <github.papyrus@mail.jlpouffier.fr>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: JLo <github.papyrus@mail.jlpouffier.fr>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: JLo <github.papyrus@mail.jlpouffier.fr>

* Update source/voice_control/assist_create_open_ai_personality.markdown

Co-authored-by: JLo <github.papyrus@mail.jlpouffier.fr>

* assist AI page - updates to review notes

* assist AI page - updates to review notes

* tiny stiles tweak

* fix capitalization. fixes the linter issue.

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>
Co-authored-by: JLo <github.papyrus@mail.jlpouffier.fr>

klaasnicolaas committed:
Update docs for EnergyZero integration (#36291)

klaasnicolaas committed:
Set VAT to true for energyzero template example (#36293)

CFenner committed:
Fix anchor in Withings docs (#36301)

klaasnicolaas committed:
Update docs for easyEnergy integration (#36292)

* Update the action parameter tables

* Add template section with examples

* Add instructions to remove the integration

* Add sections about use cases, data updates and known limitations

frenck committed:
Improve Twente Milieu documentation (#36313)

balloob committed:
Add language picker to Assist docs (#36314)

balloob committed:
Mark Spanish language as complete for Assist (#36317)

noahhusby committed:
Add removal and troubleshooting information to Russound RIO (#36299)

* Add removal, data updates, and troubleshooting information to Russound RIO

gjohansson-ST committed:
Add troubleshooting section to Nord Pool (#36302)

Co-authored-by: Simon Lamon <32477463+silamon@users.noreply.github.com>
Co-authored-by: Franck Nijhof <git@frenck.dev>

Shutgun committed:
Highlight supported devices and functions in devolo Home Control (#36297)

astrandb committed:
Add use cases etc to myuplink page (#36295)

* Add use cases etc to myuplink page

* Spelling error

* Adopt suggestions from AI

* Apply suggestions from review

klaasnicolaas committed:
Improve Autarco documentation (#36322)

dependabot[bot] committed:
build(deps): bump logger from 1.6.2 to 1.6.3 (#36330)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump nokogiri from 1.17.1 to 1.17.2 (#36328)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11690 to 0.5.11691 (#36329)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump sass-embedded from 1.82.0 to 1.83.0 (#36326)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

lukaszmn committed:
Update templating.markdown - regex_replace (#36319)

cereal2nd committed:
Velbus config parameters (#36308)

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop from 1.69.1 to 1.69.2 (#36327)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

frenck committed:
2024.12.3 (#36331)

balloob committed:
Change assist language support label (#36332)

CoMPaTech committed:
Plugwise reorder and apply config suggestions (#36087)

Co-authored-by: Klaas Schoute <klaas_schoute@hotmail.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

bdefore committed:
Update airgradient.markdown (#36338)

Add a link to the firmware page

jb101010-2 committed:
Suez_water: add removal instructions

zweckj committed:
Suez_water: add removal instructions (#36352)

tronikos committed:
Document a limitation of Google Assistant SDK (#36358)

astrandb committed:
Add an example automation (#36363)

Thank you!

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11691 to 0.5.11694 (#36371)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rexml from 3.3.9 to 3.4.0 (#36372)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump rubocop-ast from 1.36.2 to 1.37.0 (#36373)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

andrewsayre committed:
Improve HEOS docs with quality-scale items (#36336)

astrandb committed:
Address late review comments for myuplink page (#36379)

frenck committed:
Remove links to no longer existing apcupsd documentation (#36378)

frenck committed:
Fix external IP REST sensor example (#36376)

googanhiem committed:
Upload the 2024 community survey blog (#36385)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

allenporter committed:
Update Google Tasks documentation (#36365)

c0ffeeca7 committed:
Predefined sentences: timer wording (#36387)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11694 to 0.5.11699 (#36396)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

frenck committed:
2024.12.4 (#36399)

jpbede committed:
Add removal instructions to SABnzbd (#36405)

noahhusby committed:
Update high-level description for Russound RIO (#36407)

samueldowd committed:
Update smartthings.markdown (#36312)

Updated Troubleshooting section to rename Automations to Routines

laupalombi committed:
Contributing to voice initiative page creation (#36393)

* contributing to voice initiative page creation

* contributing to voice initiative page creation

* Update source/voice_control/contribute-voice.markdown

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

* Update source/voice_control/contribute-voice.markdown

* Update source/voice_control/contribute-voice.markdown

* Update source/voice_control/contribute-voice.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

* Update source/voice_control/contribute-voice.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

* linking to sidebar and tweaks in Contribute to voice initiative

---------

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

mrdarrengriffin committed:
Voice livestream card update (#36410)

* Remove current stream thumbnail

* Added new thumbnail

* Updated livestream copy

* Remove image

* Add new thumbnail

* Remove image

* Updated image

* Update news.html

googanhiem committed:
Add more tag to WTH blog (#36430)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11699 to 0.5.11704 (#36429)

Bumps [sorbet-runtime](https://github.com/sorbet/sorbet) from 0.5.11699 to 0.5.11704.
- [Release notes](https://github.com/sorbet/sorbet/releases)
- [Commits](https://github.com/sorbet/sorbet/commits)

---
updated-dependencies:
- dependency-name: sorbet-runtime
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps): bump logger from 1.6.3 to 1.6.4 (#36426)

Bumps [logger](https://github.com/ruby/logger) from 1.6.3 to 1.6.4.
- [Release notes](https://github.com/ruby/logger/releases)
- [Commits](https://github.com/ruby/logger/compare/v1.6.3...v1.6.4)

---
updated-dependencies:
- dependency-name: logger
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump json from 2.9.0 to 2.9.1 (#36427)

Bumps [json](https://github.com/ruby/json) from 2.9.0 to 2.9.1.
- [Release notes](https://github.com/ruby/json/releases)
- [Changelog](https://github.com/ruby/json/blob/master/CHANGES.md)
- [Commits](https://github.com/ruby/json/compare/v2.9.0...v2.9.1)

---
updated-dependencies:
- dependency-name: json
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump google-protobuf from 4.29.1 to 4.29.2 (#36428)

Bumps [google-protobuf](https://github.com/protocolbuffers/protobuf) from 4.29.1 to 4.29.2.
- [Release notes](https://github.com/protocolbuffers/protobuf/releases)
- [Changelog](https://github.com/protocolbuffers/protobuf/blob/main/protobuf_release.bzl)
- [Commits](https://github.com/protocolbuffers/protobuf/commits)

---
updated-dependencies:
- dependency-name: google-protobuf
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

jpbede committed:
Describe installation parameters of SABnzbd (#36441)

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

laupalombi committed:
Fixes to contribute voice page (#36440)

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

andrew-codechimp committed:
Add Mealie high level description (#36435)

* Update mealie.markdown

* Fix nitpick self-hosted

Co-authored-by: Joost Lekkerkerker <joostlek@outlook.com>

googanhiem committed:
VPE launch blog (#36443)

* Uploading blogs for Voice PE launch

* Remove old link

* Apply suggestions from code review

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

---------

Co-authored-by: Darren Griffin <darren.griffin@live.co.uk>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

mrdarrengriffin committed:
Voice Preview Edition website launch (#36442)

* Initial upload of new page

* Updated Gemfile

* Accessibility and performance improvements

* Performance optimisations

* Improve link description

* Better align video

* Updated Gemfile

* Added OG image, updated frontpage image reference

* Bumped up gradient preload

* Updated image

* Updated FAQ

* Hide AU distributors

* Video lazy load fix

* Updated distributors

* Update distributors

* Remove teaser stream card

* Add frontpage image

* Adds rootMargin to prevent lazy load flashing

mrdarrengriffin committed:
Voice Preview Edition display and content fixes (#36447)

* Fix display issues when :target is active

* Content update

* Change :target to display block

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11704 to 0.5.11707 (#36452)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

MatStace committed:
Fix broken link in expanding_assist.markdown (#36449)

frenck committed:
2024.12.5 (#36453)

gr2m committed:
docs: typo in /voice_control/assist_create_open_ai_personality (#36455)

```diff
- Pick the LLM provider of your choice, either local or cloud, as long as it las a conversational agent.
+ Pick the LLM provider of your choice, either local or cloud, as long as it has a conversational agent.
```

SNoof85 committed:
fix typo (#36470)

theofanisv committed:
fix enable_i2c.md (#36446)

tjorim committed:
Add ROBBshop to list of HA Green distributors (#36448)

ScottG489 committed:
Remove mentions of timer finished event possibly firing on startup (#36423)

Xitee1 committed:
Update voice control troubleshooting screenshots and instructions (#36483)

tr4nt0r committed:
Add configuration parameters to IronOS docs (#36366)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

domdomegg committed:
cla: handle GitHub API errors (#36489)

Jordi1990 committed:
Improve Aquacell documentation (#36486)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

regg00 committed:
added instructions to disable phone bluetooth + added debug flag to h… (#36389)

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>
Co-authored-by: Franck Nijhof <frenck@frenck.nl>

wbyoung committed:
Document ZHA global options (#34670)

squishdacat committed:
Add further information to Ness Alarm Integration regarding relevant hardware setup (#36078)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

dependabot[bot] committed:
build(deps): bump tilt from 2.4.0 to 2.5.0 (#36498)

dependabot[bot] committed:
build(deps-dev): bump prism from 1.2.0 to 1.3.0 (#36497)

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11707 to 0.5.11708 (#36496)

TheJulianJES committed:
Fix ZHA `database_path` marked as required (#36493)

TheJulianJES committed:
Remove ZHA XY color option docs (#36491)

mrdarrengriffin committed:
Updates VPE distributor list (#36514)

* Fix display issues when :target is active

* Content update

* Change :target to display block

* Added Australian distributors

andrewstech committed:
Add Monzo to the Finance category (#36510)

eblekkenhorst committed:
Update sensor.rest.markdown for external IP address sensor (#36503)

iMicknl committed:
Improve Overkiz documentation (#36505)

sorgfresser committed:
Add removal instructions to ElevenLabs (#36507)

abmantis committed:
Update description and remove section for Idasen Desk (#36523)

dsellers1 committed:
Update button.markdown (#36527)

brucedk committed:
Update alexa.smart_home.markdown to include Brazilian Portuguese (#36529)

balloob committed:
Update OpenAI default model

It's gpt-4o-mini.

Fixes #36304

tronikos committed:
Update OpenAI default model (#36532)

starkillerOG committed:
Fix styling mistake in Reolink docs (#36536)

CFenner committed:
Update vicare.markdown

CFenner committed:
Update vicare.markdown

CFenner committed:
move section

CFenner committed:
Apply suggestions from code review

Co-authored-by: Josef Zweck <josef@zweck.dev>

zweckj committed:
Add removal instructions to ViCare readme (#36547)

mj23000 committed:
Fix Bang & Olufsen limitations wording (#36550)

* Fix limitations wording

* Reorder limitations section

dependabot[bot] committed:
build(deps-dev): bump rbs from 3.7.0 to 3.8.0 (#36549)

Bumps [rbs](https://github.com/ruby/rbs) from 3.7.0 to 3.8.0.
- [Release notes](https://github.com/ruby/rbs/releases)
- [Changelog](https://github.com/ruby/rbs/blob/master/CHANGELOG.md)
- [Commits](https://github.com/ruby/rbs/compare/v3.7.0...v3.8.0)

---
updated-dependencies:
- dependency-name: rbs
  dependency-type: indirect
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
build(deps-dev): bump bigdecimal from 3.1.8 to 3.1.9 (#36548)

Bumps [bigdecimal](https://github.com/ruby/bigdecimal) from 3.1.8 to 3.1.9.
- [Release notes](https://github.com/ruby/bigdecimal/releases)
- [Changelog](https://github.com/ruby/bigdecimal/blob/master/CHANGES.md)
- [Commits](https://github.com/ruby/bigdecimal/compare/v3.1.8...v3.1.9)

---
updated-dependencies:
- dependency-name: bigdecimal
  dependency-type: indirect
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

karwosts committed:
Update lock.template.markdown (#36546)

CoMPaTech committed:
Add troubleshooting and known limitations for Plugwise (#36504)

* Add troubleshooting and known limitations

* Remove quotes for non-code

* CRAI nitpicks

* Add suggestions

Didgeridrew committed:
Update template.markdown Legacy Binary Sensor (#36542)

Example in Legacy binary sensor section was missing quote marks.

frenck committed:
Remove entity IDs from Transmission documentation (#36525)

ildar170975 committed:
Update scrape.markdown (#36556)

dependabot[bot] committed:
build(deps): bump nokogiri from 1.17.2 to 1.18.0 (#36559)

dependabot[bot] committed:
build(deps-dev): bump regexp_parser from 2.9.3 to 2.10.0 (#36558)

markhannon committed:
Add note about integration removal

markhannon committed:
Extra sections and content

markhannon commented on source/_integrations/zimi.markdown:
```diff
@@ -0,0 +1,92 @@
+
+The following Zimi devices are not yet supported:
+
+- Zimi Matter Connect ([links to specifications](https://zimi.life/product/cloud-connect/))
```
The matter connect link is under construction at the manufacturers website.   The cloud connect link has matter connect information.

dependabot[bot] committed:
build(deps-dev): bump rbs from 3.8.0 to 3.8.1 (#36566)

Commit:
Update workday.markdown (#36568)

Point to the correct repository and not to a forked one.

inthehoose committed:
Update london_underground.markdown (#36575)

Added new London Overground names

Mause committed:
docs: fix link to mosquitto addon (#36591)

johnvey committed:
Fix broken PiMusicBox link (#36585)

The previous link now points to a spam domain. PiMusicBox has been discontinued and the Github project archived.

Per the project page:

** This project is no longer maintained **
There will be no more releases and probably no more support from myself. The Pi 4 and Zero W 2 do not work with the last release and I don't know of any workarounds. Potential alternative software suggestions are available here. The software will continue to be available for now but note it is very outdated, doesn't support the latest Pi hardware and is totally unsupported. I encourage people to find alternatives.
Thanks to everyone that has helped with Pi Musicbox.

austinmroczek committed:
Improve TotalConnect docs (#36557)

* correction for Require Code

* add removal instructions

* update actions

* simplify action description

* move core notes to prerequisites

* move automation example after entity descriptions

* use consistent entity names

* consolidate limitations

* Revert "consolidate limitations"

This reverts commit aed84750ea7e45045659ab113d723b2464c0dc7a.

* consolidate limitations

* improve service unavailable

* add Data Updates section

* add Unknown to troubleshooting

* add supported devices

* add use cases

* more limitations

CoMPaTech committed:
Address supported functionality section for Plugwise (#36515)

* Initial scaffolding for supported entities

* Generalise cooling description

* Revert deletion

* Revert deletion

* Fix nitpicks

* CRAI nitpicks

* Add suggestions

* Reworded

* Improve binary

* Improve binary

joostlek committed:
Update tradfri.markdown (#36528)

CoMPaTech committed:
Improve initial section and supported devices for Plugwise (#36500)

* Incremental restart

* Fix (future and) current link endpoints

* CRAI nitpicks

* Known limitations for next PR

* Rework from 36333 data updates

* Textlint Zigbee

joostlek committed:
Fix valve names (#36606)

dependabot[bot] committed:
build(deps): bump nokogiri from 1.18.0 to 1.18.1 (#36609)

Bumps [nokogiri](https://github.com/sparklemotion/nokogiri) from 1.18.0 to 1.18.1.
- [Release notes](https://github.com/sparklemotion/nokogiri/releases)
- [Changelog](https://github.com/sparklemotion/nokogiri/blob/main/CHANGELOG.md)
- [Commits](https://github.com/sparklemotion/nokogiri/compare/v1.18.0...v1.18.1)

---
updated-dependencies:
- dependency-name: nokogiri
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

tasomaniac committed:
Fix small copy paste error (#36601)

qfoxb committed:
Tested DN15S-1BW Via MLWSB-1BW, fix link (#36630)

kylehendricks committed:
Fix typo in Google Sheets documentation (#36628)

dependabot[bot] committed:
build(deps-dev): bump ffi from 1.17.0 to 1.17.1 (#36622)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

andeart committed:
Add troubleshooting for Roomba integration (#36631)

markhannon committed:
Spelling

markhannon committed:
Merge branch 'home-assistant:current' into zimi

dependabot[bot] committed:
build(deps-dev): bump sorbet-runtime from 0.5.11708 to 0.5.11710 (#36646)

dependabot[bot] committed:
build(deps-dev): bump textlint from 14.4.0 to 14.4.2 (#36644)

saty9 committed:
Update sky_remote.markdown (#36642)

c0ffeeca7 committed:
Voice PE: fix typo in audio codec. should be AIC3204 (#36649)

andrew-codechimp committed:
Update Mastodon permissions (#36650)

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

c0ffeeca7 committed:
Wipe data disk: update steps for CM4 and CM5 (#36655)

c0ffeeca7 committed:
Add link to steps on wiping data disk on Green (#36658)

markhannon committed:
More code rabbit fixes

markhannon committed:
remove extra spaces and plus

markhannon committed:
Re-order and re-word

markhannon commented:
I just realised that I merged in lots of other development commits from other sources - should I close and re-do this PR with just the zimi content?

andrewsayre commented:
> I just realised that I merged in lots of other development commits from other sources - should I close and re-do this PR with just the zimi content?

Close and open new please. 🙂

markhannon commented:
Will close.   Replaced by https://github.com/home-assistant/home-assistant.io/pull/36665

markhannon committed:
Merge branch 'home-assistant:current' into zimi
